lefthook.yml

```diff
@@ -0,0 +1,22 @@
+# EXAMPLE USAGE:
+#
+#   Refer for explanation to following link:
+#   https://github.com/evilmartians/lefthook/blob/master/docs/configuration.md
+#
+pre-push:
+    scripts:
+        'check-for-unsuppressed-vulns.py':
+            runner: python3
+        commands:
+            audit:
+                run: yarn audit | cat
+pre-commit:
+    parallel: true
+    commands:
+        eslint:
+            glob: '*.{js,ts,jsx,tsx}'
+            run: yarn eslint --fix {staged_files}
+        prettier:
+            run: yarn prettier --write {staged_files}
```

Comment:
I think we need to add the glob to this prettier step.

I'm hitting an error on the prettier step when making a change to the yarn audit known issues file (No parser found).

Reply:
done :)

Reply:
Your original prettier PR changed the format in .json, .yml and .md files, etc. Are you sure u want to limit to these 4 files in glob?

Reply:
To be honest, I should switch this! Cheers :)